src/ComplexTypes/Address.php

```diff
@@ -85,8 +85,6 @@ class Address extends BaseType
 
     /**
      * @var string $StreetHouseNrExt
-     *     This is not used. Use separate street, house number and house number
-     *     extension parameters.
```

Comment:
This should be turned into an advice to use the separate fields if possible. Can be added to the docblock of `setStreetHouseNrExt`